src/components/Weather/WeatherTile/index.tsx

```diff
@@ -0,0 +1,137 @@
+                        <div className="weather-tile__weather-data">
+                            <UmbrellaIcon size={20} />
+                            <span className="weather-tile__weather-data--color-blue">
+                                {weather != null
+                                    ? weather[3].data.next_1_hours.details
+                                          .precipitation_amount
+                                    : '?'}
+                                <span className="weather-tile--subscript">
+                                    mm
+                                </span>
+                            </span>
+                        </div>
+                    )}
+
+                    {window.innerWidth > BREAKPOINTS.fourItems &&
+                        !(
+                            BREAKPOINTS_COMPACT.ThreeItemsDesktop <
+                                window.innerWidth &&
+                            window.innerWidth <
+                                BREAKPOINTS_COMPACT.fourItemsDesktop
+                        ) && (
+                            <div className="weather-tile__weather-data">
+                                <WindIcon size={20} />
+                                {weather != null
+                                    ? weather[3].data.instant.details.wind_speed
+                                    : '?'}
+                                <span className="weather-tile--subscript">
+                                    m/s
+                                </span>
+                            </div>
+                        )}
+                </>
+            )
+        }
+
+        return (
+            <>
+                {window.innerWidth > BREAKPOINTS.threeItems && (
+                    <div className="weather-tile__weather-data">
+                        <UmbrellaIcon size={20} />
+                        <span className="weather-tile__weather-data--color-blue">
+                            {weather != null
+                                ? weather[3].data.next_1_hours.details
+                                      .precipitation_amount
+                                : '?'}
+                            <span className="weather-tile--subscript">mm</span>
+                        </span>
+                    </div>
+                )}
+                {window.innerWidth > BREAKPOINTS.fourItems && (
+                    <div className="weather-tile__weather-data">
+                        <WindIcon size={20} />
+                        {weather != null
+                            ? weather[3].data.instant.details.wind_speed
+                            : '?'}
+                        <span className="weather-tile--subscript">m/s</span>
+                    </div>
+                )}
+            </>
+        )
+    }
+    return (
+        <div
+            className={
+                'weather-tile ' + (data.Map ? 'weather-tile-map' : 'tile')
```

Comment:
I staden for at denne komponenten har eit forhold til at det finst ein Map-visning, så burde den heller ta generelle props som gjer den gjenbrukbar. Her vil eg føreslå at WeatherTile kan ta ein `className` prop, som konsumentar av komponenten kan bruke, typ:
```
className={'weather-tile '  + props.className}
```
Då kan Map bruke den slik:
```
<WeatherTile className="weather-tile-map" />
```

Reply:
Fikk endret det til å bruke className istedet:) Fikk ikke gjort det samme med `compact`-propen. Litt usikker på hvordan jeg kan gjøre den generell når den er bassert på å bruke ulike breakpoints som er lat mtp nåværende implementasjon. Lar den stå for nå med mindre noen har et godt forslag til hvordan det kan gjøres.